tests/testthat/test-href.R

```diff
@@ -69,6 +70,7 @@ test_that("can link to local articles", {
   scoped_file_context(depth = 0)
 
   expect_equal(href_expr_(vignette("x")), "articles/y.html")
+  expect_equal(href_expr_(vignette("x", package="test")), "articles/y.html")
```

Comment:
Can you please add spaces around `=`? Otherwise it's good to merge 😄

Reply:
Gah, sorry about that. Fixed now in https://github.com/r-lib/pkgdown/pull/449/commits/228e76491a52e28b66f0dfbaa9c677a327df9e0f.